js/shop.js

```diff
@@ -1,299 +1,312 @@
+var huCourseSelection = (function (){
+  var currentEnvironment='prod';
+  var devProspectiveEnrolleeRole = '11';
```

Comment:
super nit pick, I might call this devProspectiveEnrolleeRoleId to indicate that it's an Id vs. a role name. not critical at all.